source/includes/steps-starting-compass-paste-string.yaml

```diff
@@ -10,6 +10,16 @@ content: |
   or the :manual:`DNS Seedlist Connection Format
   </reference/connection-string/#connections-dns-seedlist>`.
 
+  - To obtain the connection string from a recently used |compass-short| 
+    deployment:
+
+    a. Open |compass-short| and view list of :guilabel:`Recent` connections 
```

Comment:
[nit]

I think this reads a bit better if we use the definite article: "...and view the list of...".

Reply:
Done.